### PR TITLE
fix(config): add new firmware features to Heltun devices

### LINKED ISSUE #4094: Heltun HE-RS01 missing hold mode duration parameters & hold control mode parameter options
### Is your problem within Home Assistant (Core or Z-Wave JS Integration)?

NO, my problem is NOT within Home Assistant or the ZWave JS integration

### Is your problem within ZWaveJS2MQTT?

NO, my problem is NOT within ZWaveJS2MQTT

### Checklist

- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there.

### Describe the bug

What causes the bug?

- Device configuration files configuration for parameters 41-45 are missing options for value 4 "Momentary Toggle Switch". 
- Device configuration files are missing configuration parameters 46-50 for Hold Mode Duration (added in Firmware version 1.4)

What do you observe?

- Hold Control Mode cannot be configured as a "Momentary Toggle" as described in the manual (versions 1.0 & 1.1 for all public firmwares 1.2 thru 1.4)
- Hold Mode Duration cannot be configured (feature and parameters added in firmware 1.4 described in manual version 1.1)

What did you expect to happen?

- Hold Control Mode to have an option for "Momentary Toggle Switch" for parameters 41-45 that corresponds to a value of 4.
- Parameters 46-50 ("Hold Mode Duration") to be presented to allow configuration of Hold Mode duration for each switch.

  1. Select Heltun Relay Switch Quinto Node
  2. Scroll down to Configuration values and expand
  3. Open value drop down for any of parameters 41-45
  4. Fifth option corresponding to Momentary Toggle Switch is not present
  5. Parameters 46-50 are not listed

### Device information

Manufacturer: Heltun
Model name: Relay Switch Quinto HE-RS01
Node ID in your network: 3

### How are you using `node-zwave-js`?

- [X] `zwavejs2mqtt` Docker image (latest)
- [ ] `zwavejs2mqtt` Docker image (dev)
- [ ] `zwavejs2mqtt` Docker manually built _(please specify branches)_
- [ ] `ioBroker.zwave2` adapter _(please specify version)_
- [X] `HomeAssistant zwave_js` integration  _(please specify version)_
- [ ] `pkg`
- [ ] `node-red-contrib-zwave-js` _(please specify version, double click node to find out)_
- [ ] Manually built from GitHub _(please specify branch)_
- [ ] Other _(please describe)_

### Which branches or versions?

zwavejs2mqtt
version: 0.33.0
`node-zwave-js` branch: master
`zwavejs2mqtt` branch: master

Home Assistant wave_js integration version: 8.10.2

### Did you change anything?

no

### If yes, what did you change?

_No response_

### Did this work before?

No, it never worked anywhere

### If yes, where did it work?

_No response_

### Attach Driver Logfile

Log file was not captured when issue was presenting. My local configuration now has updated device configuration files that resolve the issues that will be submitted in a PR to fix this issue.

### PR DESCRIPTION
**Pull request by AlexCPU**
All of the manuals are available here: https://www.heltun.com/support and their changlogs detail which firmware versions they add new parameters in. :heart_eyes:

Also punted tons of parameters in to templates where common across multiple devices.

fixes: https://github.com/zwave-js/node-zwave-js/issues/4094

### CODE REVIEW

**Comment by AlexCPU** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x0344/templates/heltun_template.json [hunk 1]
```diff
@@ -192,17 +192,209 @@
...
+			{
+				"label": "Fahrenheit",
+				"value": 0
+			},
+			{
+				"label": "Celsius",
+				"value": 1
+			}
+		]
+	},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0344/templates/heltun_template.json]: Please check the master template if this parameter exists there already.

File: packages/config/config/devices/0x0344/he-hls01.json [hunk 2]
```diff
@@ -436,6 +201,27 @@
 		},
 		{
 			"#": "156",
+			"$if": "firmwareVersion >= 1.5",
+			"label": "Over-Voltage Protection & Notification",
+			"unit": "Volts",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0344/he-hls01.json]:
```suggestion
			"unit": "V",
```

Comment (AlCalzone) [reply packages/config/config/devices/0x0344/he-hls01.json]: please also update the other parameters using this unit

File: packages/config/config/devices/0x0344/he-ht01.json [hunk 3]
```diff
@@ -265,21 +194,13 @@
 		},
 		{
 			"#": "16",
-			"label": "Floor Temperature Calibration",
-			"unit": "0.1 °C",
-			"valueSize": 1,
-			"minValue": -100,
-			"maxValue": 100,
-			"defaultValue": 0
+			"$import": "templates/heltun_template.json#temp_sensor_cal",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0344/he-ht01.json]: I'd prefer if the template keys weren't abbreviated too much. --> `temp_sensor_calibration`

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Nice! A few minor changes please.

File: packages/config/config/devices/0x0344/templates/heltun_template.json (hunk 1 above)
Comment (AlexCPU) [reply packages/config/config/devices/0x0344/templates/heltun_template.json]: apparently I'm also blind...

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**